### PR TITLE
[#9766] Instructor: Sessions: Add new feedback session form should be hidden by default

### LINKED ISSUE #9766: Instructor: Sessions: Add new feedback session form should be hidden by default
- **Environment**: 
`master` branch at commit 4346f6f3d5ef8bf9570ab088a9835709f13ca99f

**Description of feature/enhancement**
Hide the **New feedback session form** by default when navigating to the sessions page as an instructor. Perhaps a card header or button can be placed instead which expands to the form when clicked.

**Justification**

Currently, navigating to the sessions page will give us this:
image. The user has to scroll down one whole page just to get to the existing feedback sessions.

When a tab is labelled "Sessions", the main intent of the user to go there should be to view the list of sessions.

### PR DESCRIPTION
**Pull request by erjillsison**
Fixes #9766

Form is collapsed by default when viewed by clicking the "Sessions" tab from the nav bar.

When navigating to this page from the home page by clicking "Add Session", the form is expanded automatically.

AddNewSessin

### CODE REVIEW

**Comment by erjillsison** [comment]
Ready for review

**Comment by crizyli** [comment]
LGTM!

File: src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html [hunk 1]
```diff
@@ -2,9 +2,20 @@
   You have not created any courses yet, or you have no active courses. Go <a href="#" (click)="createOrUnarchiveCourse(); $event.preventDefault()">here</a> to create or unarchive a course.
 </div>
 
-<tm-session-edit-form [formMode]="SessionEditFormMode.ADD" [(model)]="sessionEditFormModel"
-                      [courseCandidates]="courseCandidates" [templateSessions]="templateSessions"
-                      (addNewSessionEvent)="addNewSessionHandler()" (copyOtherSessionsEvent)="copyFromOtherSessionsHandler()"></tm-session-edit-form>
+<div>
+  <button class="btn btn-primary" (click)="isSessionEditFormExpanded=true"
```
Comment (xpdavid) [inline src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html]:
```suggestion
  <button class="btn btn-primary" (click)="isSessionEditFormExpanded = true"
```

Mind the whitespaces.

File: src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html [hunk 2]
```diff
@@ -2,9 +2,20 @@
...
+          [disabled]="isSessionEditFormExpanded"><i class="fas fa-plus"></i> Add New Feedback Session
+  </button>
+</div>
+
+<div class="margin-top-30px" *ngIf="isSessionEditFormExpanded">
+  <tm-session-edit-form
+                        [formMode]="SessionEditFormMode.ADD" [(model)]="sessionEditFormModel"
+                        [courseCandidates]="courseCandidates" [templateSessions]="templateSessions"
+                        (addNewSessionEvent)="addNewSessionHandler()" (copyOtherSessionsEvent)="copyFromOtherSessionsHandler()"
+                        (closeEditFormEvent)="isSessionEditFormExpanded=false"
```
Comment (xpdavid) [inline src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html]:
```suggestion
                        (closeEditFormEvent)="isSessionEditFormExpanded = false"
```

**Pull request merged**